### PR TITLE
spectacle: add page

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** latest

### CODE REVIEW

File: pages/common/spectacle.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# spectacle
+
+> KDE's screenshot utility.
+> More information: <https://manned.org/spectacle>.
+
+- Capture the entire desktop:
```
Comment (kbdharun) [inline pages/common/spectacle.md:6]:
```suggestion
- Capture the screenshot of the entire desktop:
```

File: pages/common/spectacle.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+`spectacle`
+
+- Capture an active window:
```
Comment (kbdharun) [inline pages/common/spectacle.md:10]:
```suggestion
- Capture the screenshot of an active window:
```

File: pages/common/spectacle.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+`spectacle --activewindow`
+
+- Capture a specific region:
```
Comment (kbdharun) [inline pages/common/spectacle.md:14]:
```suggestion
- Capture the screenshot of a specific region:
```

**Review by kbdharun — APPROVED** [review]
Thanks for your contribution. How about this?

File: pages/common/spectacle.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# spectacle
+
+> KDE's screenshot utility.
+> More information: <https://manned.org/spectacle>.
+
+- Capture a screenshot of the entire desktop:
+
+`spectacle`
+
+- Capture a screenshot of an active window:
```
Comment (pixelcmtd) [inline pages/common/spectacle.md:10]:
```suggestion
- Capture a screenshot of the active window:
```

File: pages/common/spectacle.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# spectacle
+
+> KDE's screenshot utility.
+> More information: <https://manned.org/spectacle>.
```
Comment (navarroaxel) [inline pages/common/spectacle.md:4]: I think the author's web is better
```suggestion
> More information: <https://apps.kde.org/spectacle/>.
```

Comment (kbdharun) [reply pages/common/spectacle.md:4]: Generally, We can use KDE docs for most commands. But there is no documentation on flags in https://docs.kde.org/stable5/en/spectacle/spectacle/index.html or the link you suggested in this case, I think we can either link to manned or to docs but it doesn't have any info on command line arguments.

Comment (blueskyson) [reply pages/common/spectacle.md:4]: I like the man page from manned.org more.

Comment (EmilySeville7cfg) [reply pages/common/spectacle.md:4]: - I think that we should always use the "closest" page to written examples: the page which has all used flags mentioned in examples at the bare minimum level and maybe even more.
- Also I suggest check whether such page is actively supported: prefer the page with the most recent changes.

**Review by pixelcmtd — APPROVED** [review]
(once the other review has been resolved)

File: pages/common/spectacle.md (hunk 5 above)
Comment (EmilySeville7cfg) [reply pages/common/spectacle.md:4]: I guess this conversation can be marked as resolved as PR was approved.

**Pull request merged**